ci/codecov.sh

```diff
@@ -55,11 +55,11 @@ elif [[ "$coverage_action" == "collect" ]] || [[ "$coverage_action" == "upload"
     fi
 
     # install the latest lcov we know works
-    : "${LCOV_VERSION:=v1.15}"
+    : "${LCOV_VERSION:=v2.1}"
 
     if [[ "$LCOV_VERSION" =~ ^v[2-9] ]]; then
         sudo apt-get -o Acquire::Retries="${NET_RETRY_COUNT:-3}" -y -q --no-install-suggests --no-install-recommends install libcapture-tiny-perl libdatetime-perl || true
-        LCOV_OPTIONS="${LCOV_OPTIONS} --ignore-errors unused"
+        LCOV_OPTIONS="${LCOV_OPTIONS} --ignore-errors inconsistent,mismatch,unused"
```

Comment:
What are those errors? Maybe a comment here is a good idea to avoid us having to look it up again.